OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1635,6 +1635,21 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					}
 				}
 
+				if (engineVersion < 20180225)
```

Comment:
Needs an update on this due to new release (?).

Reply:
Yeah, but it's not the only case, I think. I'll fix it either way.